### PR TITLE
Add blog post for p2pool release

### PR DESCRIPTION
**Pull request by QuickBASIC**
P2Pool binaries have been released here: 
https://github.com/SChernykh/p2pool/releases/tag/v1.0
Source: 
https://github.com/SChernykh/p2pool

### CODE REVIEW

**Comment by erciccione** [comment]
@SChernykh could you review this PR?

Tracked in #1802

File: _posts/2021-09-18-p2pool-released.md
Comment (erciccione) [inline _posts/2021-09-18-p2pool-released.md:4]:
```suggestion
summary: P2Pool, the decentralized pool for Monero mining, is live on mainnet

```

File: _posts/2021-09-18-p2pool-released.md
Comment (erciccione) [inline _posts/2021-09-18-p2pool-released.md:23]: > solving exchange delisting/KYC threat

This is not really the case. it helps for sure, but the problem remains for xmr <-> fiat and you cannot do atomic swaps with that.

File: _posts/2021-09-18-p2pool-released.md
Comment (erciccione) [inline _posts/2021-09-18-p2pool-released.md:13]:
```suggestion
Monero P2Pool is a peer-to-peer Monero mining pool developed from scratch by SChernykh (also known as sech1). P2Pool combines the advantages of pool and solo @mining; you still fully control your Monero @node and what it mines, but you get frequent payouts like on a regular pool. P2Pool as a concept was first developed for the Bitcoin blockchain but was never fully realized due to certain limitations, mainly because it had a problem with orphaned blocks which is solved in Monero P2Pool by uncle blocks.

```

File: _posts/2021-09-18-p2pool-released.md
Comment (SChernykh) [inline _posts/2021-09-18-p2pool-released.md:19]: Change 0.0005 to 0.0004 to have the same number as further down in the text. 0.0004 XMR is the correct number.

File: _posts/2021-09-18-p2pool-released.md
Comment (SChernykh) [inline _posts/2021-09-18-p2pool-released.md:49]: Maybe add a link to https://p2pool.io/ here? WIth a note that it's not a central pool server, just a monitoring page.

Comment (SChernykh) [reply _posts/2021-09-18-p2pool-released.md:49]: Also, since it's a 0% fee pool, having my donation address in the article would be appreciated: 44MnN1f3Eto8DZYUWuE5XZNUtE3vcRzt2j6PzqWpPau34e6Cf4fAxt6X2MBmrm6F9YMEiMNjN6W4Shn4pLcfNAja621jwyg

File: _posts/2021-09-18-p2pool-released.md
Comment (erciccione) [inline _posts/2021-09-18-p2pool-released.md:27]:
```suggestion
P2Pool is a sidechain to Monero and P2Pool blocks are potentially Monero @blocks. Each miner submits block templates that include a payout for all of the miners that currently have shares in the PPLNS window. High quality block templates are added to the P2Pool blockchain as blocks which count as "shares" for the miner who found them. If a block is good enough to be a Monero block it is also submitted to the Monero network to be included in its blockchain. Once the Monero block is confirmed by the network, those miners are directly paid in the @coinbase-transaction because they were included already in the block. If P2Pool "shares" are found a the same block height as an existing share, it is still included as an uncle block (worth 20% less than a normal share) so you can still get paid for it. (Uncle blocks can be submitted up to 3 blocks behind the current height and still be included.)

```

File: _posts/2021-09-18-p2pool-released.md
Comment (erciccione) [inline _posts/2021-09-18-p2pool-released.md:9]: Just to change from "mainnet" that is used multiple times. This is clearer for less technical people who don't know what mainnet is.
```suggestion
We are happy to announce that Monero P2Pool is live on the Monero network and binaries have been [released](https://github.com/SChernykh/p2pool/releases).

```

File: _posts/2021-09-18-p2pool-released.md
Comment (SChernykh) [reply _posts/2021-09-18-p2pool-released.md:23]: Indirectly it still solves it. `xmr <-> btc <-> fiat` will always be possible with atomic swaps.

Comment (erciccione) [reply _posts/2021-09-18-p2pool-released.md:23]: That's already the case, not necessarily with atomic swaps. Anyway, it's not really important. If people prefer this phrasing i'm ok with it.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:19]: Ack. Fixed.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:4]: Lol. That's why we review these things. Not sure how that slipped in.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:49]: Ack. Added and added.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:23]: This verbiage was actually toned down from the original text (from SChernykh). I'll leave it for now and if anyone has a direct suggestion of how to change it we can do so.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:13]: Ack. Links added.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:27]: Ack.

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:9]: Ack.

File: _posts/2021-09-18-p2pool-released.md
Comment (erciccione) [inline _posts/2021-09-18-p2pool-released.md:4]: There is an added typo now (the the). I [suggested](https://github.com/monero-project/monero-site/pull/1835#discussion_r711699621):
```suggestion
summary: P2Pool, the decentralized pool for Monero mining, is live on mainnet

```

**Review by erciccione — CHANGES_REQUESTED** [review]

File: _posts/2021-09-18-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md]: Ack. Fixed.

File: _posts/2021-09-18-p2pool-released.md
Comment (hyc) [inline _posts/2021-09-18-p2pool-released.md:27]: For the sake of editiability and line-based references, you should break up long lines like this into shorter chunks.
Commenting here because you need `s/found a the same/found at the same/` on this "line" ...

Comment (QuickBASIC) [reply _posts/2021-09-18-p2pool-released.md:27]: Ack. Fixing. I forgot markdown ignores linebreaks when formatting unless there's two in a row for paragraph end.

**Comment by QuickBASIC** [comment]
Should I bump the date on this since it's already 3 days passed? Will it mess up the ordering since there's already an interim post (i.e. RSS feed, etc).

**Comment by erciccione** [comment]
I would wait at least a week before merging this blog post (with the date changed). Better not publish many posts at the same time.

**Comment by QuickBASIC** [comment]
Understood. I'll update the date and then it can be merged any time after 2021-09-28.

**Comment by QuickBASIC** [comment]
Any possibility of this being merged?

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:3]: drop ""

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:16]: this is a stylistic change, but I recommend a semicolon over `---` for reasons clearer on line 20

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:20]: having a period followed by `---` is unusual. consider replacing with `format; each`

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:24]: "Since atomic swaps are now live (which address the exchange delisting/KYC threat),"

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:25]: Replace semicolon with period and remove comma before `because a pool`

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:29]: comma before `and`

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:33]: remove first `still` in `still included`.

remove period after `paid for it.`, lowercase U in `Uncle`, and move period outside of parenthesis to be `included).`

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:39]: replace comma with semicolon or add `and` after comma

File: _posts/2021-09-28-p2pool-released.md
Comment (SamsungGalaxyPlayer) [inline _posts/2021-09-28-p2pool-released.md:56]: also reference Matrix

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]
Some of these changes may seem minor, but it currently reads strangely to me as-is. The quotes need to be dropped to build properly I think (I don't see the blog post in the preview).

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:56]: I don't use matrix, what link should I include?

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:3]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:16]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:20]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:24]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:25]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:33]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:29]: Ack

File: _posts/2021-09-28-p2pool-released.md
Comment (QuickBASIC) [reply _posts/2021-09-28-p2pool-released.md:39]: Ack

**Comment by QuickBASIC** [comment]
@SamsungGalaxyPlayer Fixed per your suggestions. Thank you. I'm an overzealous comma user. :-P

**Comment by SamsungGalaxyPlayer** [comment]
Okay, someone else will have to confirm the build works now, but thanks :)

**Comment by erciccione** [comment]
> The quotes need to be dropped to build properly I think 

It will build regardless.

@QuickBASIC please squash your commits into 1 so we can make a final review.

**Comment by QuickBASIC** [comment]
@erciccione The squashes have been squished and the commits are squoshed.

**Review by erciccione — APPROVED** [review]
Looks good to me. There are some technical info, so would be probably a good idea to have some links to the Moneropedia entries for complicated teminology (like coinbase transactions, blocks, transactions, etc). It would be a nice to have,but not a blocker. I will add the snippet necessary to have links working to the readme, because would be a good idea to always use those links when useful.

**Comment by QuickBASIC** [comment]
We added those links previously and then they were removed because we couldn't confirm at that time that they were working. I could have sworn someone said there was some change that needed to be made for them to work on blog posts. I could re-add the one's you suggested in your first review if you're sure they'll work, but I'd really not prefer to delay merging this for the links honestly.

**Comment by erciccione** [comment]
> We added those links previously and then they were removed because we couldn't confirm at that time that they were working

This is not entirely correct. You were asked to add links here and you acked (https://github.com/monero-project/monero-site/pull/1835#discussion_r711700543 and https://github.com/monero-project/monero-site/pull/1835#discussion_r711700833) the suggestion and marked it as resolved, then you said on IRC that links weren't working, so i told you to add a snippet (as in #1838) to make the links work. Instead of doing that you removed the links entirely without mentioning it anywhere.

In this case the suggestions were not deal breakers, so as i said, not a stopper, and i can add them later on, but please make sure to follow the expected behaviour for collaborative development and review in the future :)

**Comment by QuickBASIC** [comment]
@erciccione Sorry for the misunderstanding. I'm not sure what happened here (probably lack of coffee when I was updating it). I've updated the PR with the suggested changes to make moneropedia links work and tested locally.

**Review by erciccione — APPROVED** [review]
No worries! :)

**Comment by erciccione** [comment]
Now that [p2pool has released a version that works with the latest Monero binaries](https://github.com/SChernykh/p2pool/releases/tag/v1.1), we should edit this blog post to reflect that.

**Comment by SChernykh** [comment]
@erciccione Not the latest released, but compiled from the latest code.

**Comment by erciccione** [comment]
Ah i see. I misunderstood. In that case we can merge.

**Pull request merged**
